pandas/tests/libs/test_hashtable.py

```diff
@@ -80,6 +82,8 @@ def test_map(self, table_type, dtype):
             table = table_type()
             keys = np.arange(N).astype(dtype)
             vals = np.arange(N).astype(np.int64) + N
+            keys.flags.writeable = False
```

Comment:
can you parameterize all of these on writeable True/False

Reply:
Done: https://github.com/pandas-dev/pandas/pull/38179/commits/073222ee160b2d7060a317b5e71bc27c027ec8b1